Zend/zend_generators.c

```diff
@@ -330,6 +325,10 @@ static void zend_generator_free_storage(zend_object *object) /* {{{ */
 
 	zend_generator_close(generator, 0);
 
+	if (generator->func && generator->func->common.fn_flags & ZEND_ACC_CLOSURE) {
```

Comment:
Nit: We use braces around `&` expressions, even if precedence is correct.

Reply:
It turns out that the check for `generator->func` was not necessary anyways (all tests pass and it's unconditionally accessed in other locations), thus fixing it that way.

Reply:
As the CI (and Dmitry) pointed out, that comment was not correct. I've now just added the parentheses.